src/rgw/rgw_rest_swift.cc

```diff
@@ -652,15 +652,13 @@ static int get_delete_at_param(req_state *s, real_time *delete_at)
 
   if (x_delete.empty()) {
     x_delete = s->info.env->get("HTTP_X_DELETE_AT", "");
+    return 0;
```

Comment:
@yehudasa Thank for you reminding me.  I think I made a mistake, and the code had been rolled back.